src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -82,10 +82,24 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
         val result = driver.executeScript("""
             try {
                 window._jibriParticipants = [];
+                const existingMembers = APP.conference._room.room.members || {};
+                console.log("There were " + Object.keys(existingMembers).length + " existing members");
+                for (let key in existingMembers) {
+                    const existingMember = existingMembers[key];
+                    if (existingMember.identity) {
+                        console.log("Member ", existingMember, " has identity, adding");
+                        window._jibriParticipants.push(existingMember.identity);
+                    } else {
+                        console.log("Member ", existingMember.jid, " has no identity, skipping");
+                    }
+                }
```

Comment:
Do you have to take into account any "hidden" members, like the focus?

Reply:
from what I found, those won't have 'identity' so we'll be covered by the check later.